### ISSUE TITLE
[ic-status-tag] outlined variant doesn't have sufficient contrast in HCM

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#3930]
### Summary of the bug 
In high contrast mode, the default status tag appears correctly but the outlined variant doesn't have sufficient colour contrast.

Not sure if it's just the text colour that needs updating or the background colour too.

### 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to status tag error, warning or success
2. Change to high contrast mode (forced colors active in web browser)
3. See the error for both light and dark mode.

### 📸 Screenshots or code 

<img width="210" height="70" alt="Image" src="https://github.com/user-attachments/assets/7cde23b6-3337-4b1b-b418-f79c1e4897b5" />
<img width="236" height="76" alt="Image" src="https://github.com/user-attachments/assets/7bb769f9-db15-4612-8d45-ca5620c4fd8b" />
<img width="196" height="73" alt="Image" src="https://github.com/user-attachments/assets/415d9cbb-33fa-474a-9247-8b149258fe1a" />
<img width="224" height="64" alt="Image" src="https://github.com/user-attachments/assets/f6b7c6db-5bef-47d9-9380-c90c16ca8f15" />
<img width="218" height="58" alt="Image" src="https://github.com/user-attachments/assets/6375e6db-0e9a-41b4-a9b8-01405cb148f9" />

### 🧐 Expected behaviour 
Status tag background and text should be visible.
